src/main/webapp/templates/dialog.hbs

```diff
@@ -1,19 +1,31 @@
 <dialog>
     <div class="container">
         <div class="header">
+            {{#if headerError}}
+            <div class="headerError"><img src="../img/exclamation_mark.svg" />{{headerError}}</div>
+            {{/if}}
             <div class="label">
                 {{actionText}} <a href="{{url}}">{{key}}</a>
             </div>
             <h1>{{{subject}}}</h1>
+            <div class="label key-value">
+                <span class="key">Assignee</span>
```

Comment:
According to the proposed design, this key should be 'Current Assignee'.

Reply:
Fixed!